webnn/resources/utils.js

```diff
@@ -94,14 +95,20 @@ const getConv2dPrecisionTolerance = (resources, operationName) => {
       inputChannels = options.inputLayout === 'nchw' ? inputChannels : inputShape[3];
     }
     if (options.filterLayout) {
-      if (!['oihw', 'hwio', 'ohwi', 'ihwo'].includes(options.filterLayout)) {
+      let filterLayouts = ['oihw', 'hwio', 'ohwi', 'ihwo']; // default for conv2d
```

Comment:
Nit: s/let/const

Reply:
@Honry This variable `filterLayouts` will be set to a new value from next Line-99 as following:
```js
      if (operationName === 'convTranspose2d') {
        filterLayouts = ['iohw', 'hwoi', 'ohwi'];
      }
```
PTAL, thanks.